syne_tune/optimizer/schedulers/searchers/bayesopt/gpautograd/independent/likelihood.py

```diff
@@ -55,12 +55,26 @@
 
 class IndependentGPPerResourceMarginalLikelihood(MarginalLikelihood):
     """
-    Marginal likelihood for GP multi-fidelity model over f(x, r), where for
-    each r, f(x, r) is represented by an independent GP. The different
-    processes share the same kernel, but have their own mean functions mu_r
-    and covariance scales c_r.
-    If `separate_noise_variances` is True, each process has its own noise
+    Marginal likelihood for GP multi-fidelity model over :msth:`f(x, r)`,
```

Comment:
Fix typo: `msth` -> `math` currently in main: https://github.com/awslabs/syne-tune/blob/a970a9073ba916515e90c225080040da4756443a/syne_tune/optimizer/schedulers/searchers/bayesopt/gpautograd/independent/likelihood.py#L58

```suggestion
    Marginal likelihood for GP multi-fidelity model over :math:`f(x, r)`,
```

Reply:
Thanks @ConnorBaker . https://github.com/awslabs/syne-tune/pull/472